db/schema.rb

```diff
@@ -281,6 +273,28 @@
     t.index ["phone_number"], name: "index_channel_whatsapp_on_phone_number", unique: true
   end
 
+  create_table "companies", force: :cascade do |t|
```

Comment:
We don't have companies table yet, we might need another PR to clean this up  😆

Reply:
@pranavrajs Ohh is it my changes? Let me check this. :D